foodpairing/inspire_kg
# CFI-3496: Inspire KG technical documentation

This PR contains only the `README.md` that contains a handful of practical use cases for the Inspire KG, in the form of SPARQL queries. 

In order to test the queries out:
1. Go to our [GraphDB instance on staging](https://graphdb.internal.staging.foodpairing.com/). 
2. Make sure the `inspire` repository is selected (top-right corner).
3. Go to the SPARQL tab.
4. Run the queries (i.e., copy-paste the SPARQL excerpts and press `RUN`) and verify whether a result-set is retrieved.
5. Suggest additional SPARQL queries that may be illustrative enough for potential end users.
